poh/src/poh_recorder.rs

```diff
@@ -539,20 +538,19 @@ impl PohRecorder {
     }
 
     pub fn tick(&mut self) {
-        let now = Instant::now();
         let (poh_entry, target_time) = {
+            let now = Instant::now();
```

Comment:
while we're here, should we be using `solana_measure` to be consistent with the rest of the codebase?